### PR TITLE
Multiple code improvements - squid:S1213, common-java:DuplicatedBlocks, squid:S1118, squid:MethodCyclomaticComplexity, squid:SwitchLastCaseIsDefaultCheck, squid:S1192, squid:S1301, squid:S1488

### PR DESCRIPTION
**Pull request by georgekankava**
This pull request is focused on resolving occurrences of Sonar rules
squid:S1213 - Standard outputs should not be used directly to log anything.
common-java:DuplicatedBlocks - Standard outputs should not be used directly to log anything.
squid:S1118 - Standard outputs should not be used directly to log anything.
squid:MethodCyclomaticComplexity - Standard outputs should not be used directly to log anything.
squid:SwitchLastCaseIsDefaultCheck - Standard outputs should not be used directly to log anything.
squid:S1192 - Standard outputs should not be used directly to log anything.
squid:S1301 - Standard outputs should not be used directly to log anything.
squid:S1488 - Standard outputs should not be used directly to log anything.
You can find more information about the issue here:
https://dev.eclipse.org/sonar/rules/show/squid:S1213
https://dev.eclipse.org/sonar/rules/show/common-java:DuplicatedBlocks
https://dev.eclipse.org/sonar/rules/show/squid:S1118
https://dev.eclipse.org/sonar/rules/show/squid:MethodCyclomaticComplexity
https://dev.eclipse.org/sonar/rules/show/squid:SwitchLastCaseIsDefaultCheck
https://dev.eclipse.org/sonar/rules/show/squid:S1192
https://dev.eclipse.org/sonar/rules/show/squid:S1301
https://dev.eclipse.org/sonar/rules/show/squid:S1488
Please let me know if you have any questions.
George Kankava

### CODE REVIEW

File: junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/BaseModel.java [hunk 1]
```diff
@@ -6,9 +6,11 @@
 import javax.xml.bind.annotation.XmlRootElement;
 import javax.xml.bind.annotation.XmlValue;
 
-@XmlRootElement(name = "failure")
+/**
+ * Created by georgekankava on 5/5/16.
+ */
```
Comment (BogdanLivadariu) [inline junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/BaseModel.java]: for consistency reasons, please remove the author section
as all the source files do not include this sort of info.

Comment (georgekankava) [inline junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/BaseModel.java]: @BogdanLivadariu Sure, sorry about that. IDE generated.

Comment (BogdanLivadariu) [inline junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/BaseModel.java]: I know the IDE likes to be sticky :)

Comment (georgekankava) [inline junit-reporting-handlebars/src/main/java/com/github/bogdanlivadariu/reporting/junit/xml/models/BaseModel.java]: Done.

**Comment by BogdanLivadariu** [comment]
👍

**Pull request merged**